resources/assets/less/bem/page-extra.less

```diff
@@ -97,5 +119,9 @@
     right: 0;
     top: 0;
     cursor: move;
+
+    .@{top}--userpage & {
+      margin-right: 30px;
```

Comment:
Apparently the margin is different between mobile and desktop. Also both have their own variables.